### DISCUSSION TITLE
Creating Supplementary Figure 1,2,... in Quarto

### DISCUSSION THREAD

**Discussion by JauntyJJS** [quarto-dev/quarto-cli#3209] [Q&A] [answered]
Hi,

I tried to follow the instructions indicated in https://quarto.org/docs/authoring/figures.html to create figures in a word document.

However, instead of Figure 1 highlighted, is it possible to change it to Supplementary Figure 1 instead ?

![image](https://user-images.githubusercontent.com/9066508/200288912-d891c4fe-ea5a-4996-b349-3d673c23dff7.png)

The code written are as follows:

```
---
title: "Figure_Issue"
format: docx
---

::: {#fig-avatar layout-ncol=1}

![](image/avatar.jpg)

Famous Avatar
:::

@fig-avatar describes 

```

Here is the attached file for my code and images

[figure_issue.zip](https://github.com/quarto-dev/quarto-cli/files/9950583/figure_issue.zip)

Thank you.

**Comment by dragonstyle** [comment, marked as answer]
You should be able to Accomplish this like so:

```markdown
---
title: "Figure_Issue"
format: docx
crossref:
  fig-title: Supplementary Figure
  fig-prefix: Sup Fig
---

[figure_issue.zip](https://github.com/quarto-dev/quarto-cli/files/9950580/figure_issue.zip)

::: {#fig-avatar layout-ncol=1}

![](image/avatar.jpg)

Famous Avatar
:::

@fig-avatar describes 
```

More about the options here:

https://quarto.org/docs/authoring/cross-references.html#options

**Reply by JauntyJJS** [reply]
Thank you

**Comment by bbartholdy** [comment]
I wanted to reopen this discussion because I personally prefer labelling supplementary figures as 'Figure S1', 'Figure S2', etc. I currently don't see a way to do this @dragonstyle ?

The closest I could get was `fig-title: Figure S` which results in 'Figure S 1' (not ideal)

**Reply by dragonstyle** [reply]
I don't currently see a way to accomplish this - we currently always place a non-breaking space between the prefix and number. This would be a nice enhancement to support and @cscheid is planning to do work in this area for Quarto 1.4. Would you mind opening an enhancement request?

**Reply by bbartholdy** [reply]
Sure! Via Issues or Discussions > Feature Requests?

**Reply by dragonstyle** [reply]
Issues would great, ty!

**Comment by slowkow** [comment]
Could I ask if this feature is still in-progress or if there is a way we can do this?

Specifically, I'm looking to have two separate sets of figures:

- The first set of figures should be numbered like `Figure 1`, ...

- The second set of figures should be numbered like `Figure S1`, ...

Both sets of figures are referenced in the same document. Of course, I could make two separate documents and then try to stitch them together at the end. But I'm hoping there is a less laborious path forward.

By the way, if there is any way to write extensions or plugins for quarto, I'd appreciate if anyone would share your favorite link that describes how to get started on writing and using an extension.

Also, it'd be great if this discussion had a link to the relevant GitHub Issue, because I arrived here via a search engine.

**Reply by mcanouil** [reply]
See <https://quarto.org/docs/authoring/cross-references-custom.html>.

Reading the documentation should get you started: <https://quarto.org/docs/extensions/>.

**Reply by slowkow** [reply]
Thanks, the custom cross references link is exactly what I was looking for.

**Reply by slowkow** [reply]
Here is the relevant GitHub issue: https://github.com/quarto-dev/quarto-cli/issues/1655
